packages/zosjobs/src/DownloadJobs.ts

```diff
@@ -101,8 +104,21 @@ export class DownloadJobs {
         this.log.trace("Entering downloadSpoolContentCommon with parms %s", JSON.stringify(parms));
         ImperativeExpect.keysToBeDefined(parms, ["jobFile"], "You must specify a job file on your 'parms' parameter" +
             " object to the downloadSpoolContentCommon API.");
-        const job = parms.jobFile;
 
+        //waiting for job to be active before continuing with job download
+        if (parms.waitForActive) {
+            await MonitorJobs.waitForActiveStatus(session, parms.jobname, parms.jobid);
+        }
+
+        //waiting for job status to be output before continuing on with job download
+        if (parms.waitForOutput) {
+            await MonitorJobs.waitForJobOutputStatus(session, {
+                jobname: parms.jobname,
+                jobid: parms.jobid
+            } as IJob);
+        }
```

Comment:
Curious if adding the new `waitForActiveStatus` method is necessary? It may be possible to combine these two `if` blocks into a single one that calls `waitForStatusCommon`.

Feel free to disregard this comment as I think the current code behaves correctly, so it's really just a stylistic preference 😋 

```suggestion
        //waiting for job status to change before continuing with job download
        if (parms.waitForActive || parms.waitForOutput) {
            await MonitorJobs.waitForStatusCommon(session, {
                jobname: parms.jobname,
                jobid: parms.jobid,
                status: parms.waitForActive ? JOB_STATUS.ACTIVE : JOB_STATUS.OUTPUT;
            });
        }
```